pisa/stages/data/sqlite_loader.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+import pandas as pd
+import sqlite3
+from pisa import FTYPE
+from pisa.core.stage import Stage
+from pisa.utils import vectorizer
+from pisa.utils.resources import find_resource
+from pisa.utils.profiler import profile
+from pisa.core.container import Container
+
+
+class sqlite_loader(Stage):
+    """
+    SQLite loader PISA Pi class
+    Parameters
+    ----------
+    database : path to sqlite database
+    **kwargs
+        Passed to Stage
+    """
+    def __init__(
+        self,
+        database,
+        output_names,
+        add_these =  ['energy', 'zenith'],
```

Comment:
could it simply be `coszen` here? Because right now you specify to add `zenith` but in the code then change this case to `coszen`, which is a bit convoluted and unexpected behaviour

Reply:
I'll rewrite this!